### PR TITLE
Doc update: add the debugging and property desc

### PR DESCRIPTION
**Pull request by ianull**
Add debugging and inspection instructions and description of
default Arcs properties.

### CODE REVIEW

File: javaharness/README.md [hunk 1]
```diff
@@ -39,3 +39,49 @@ and everything that needs to be rebuilt should be.
   ```bash
   bazel clean --expunge
   ```
+
+## Debugging and Inspection
+The Arcs Local Development Server (ALDS) is used to proxy message channels between<br />
```
Comment (csilvestrini) [inline javaharness/README.md]: These `<br />` tags don't seem necessary.

Comment (csilvestrini) [inline javaharness/README.md]: Nit: "message channels" -> "messages"

Comment (ianull) [reply javaharness/README.md]: Thanks! Removed them.

I used to add '\' or '<br />' or two trailing spaces to force line break.
Without any of these magics, some markdown editors cannot present results correctly.
But they are unnecessary in GH :)

Comment (ianull) [reply javaharness/README.md]: Np, changed it!

**Pull request merged**